util/app-config/src/app_config.rs

```diff
@@ -12,6 +12,7 @@ use serde::{Deserialize, Serialize};
 
 use ckb_chain_spec::ChainSpec;
 use ckb_logger_service::Config as LogConfig;
+use ckb_metrics_service::Config as MetricsConfig;
```

Comment:
It's better to create a config crate so that app config does not  depend on services. The same is the logger config.

Reply:
DONE!